rm2/src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./components/App";
+
+ReactDOM.render(<App />, document.getElementById("app"));
```

Comment:
Consider moving all the "rm2" folder code up to the root, outside of the folder, so that the repo just had 1 project.
"rm1" from the 1st PR could be just be one of the folders in our project.
The reasoning is that we will be applying changes to the same code going forward so there's no point in keeping it nested instead of at the root.

Reply:
Yes, I will remove it. so i created the different directory since i has to builds this from ground up without create-react-app. I wanted to note the step.

Reply:
Sure thing, that's a solid reason!
Let's move everything up then, to avoid doing that later.